linkerd/app/integration/src/tests/telemetry.rs

```diff
@@ -1170,8 +1164,7 @@ mod transport {
             metrics::labels()
                 .label("authority", proxy.outbound_server.as_ref().unwrap().addr)
                 .label("direction", "outbound")
-                .label("tls", "no_identity")
-                .label("no_tls_reason", "not_provided_by_service_discovery")
+                .label("tls", "disabled")
```

Comment:
Nit/TIOLI: now that we use the same labels in the inbound and outbound directions, rather than `tls="disabled"` on inbound and `tls="no_identity"` on outbound, we could move this label out of the per-direction labels and into the common labels built in the test function,

Reply:
do you mean adding this to `Labels::default()`? In any case, I'll save that for a followup.

Reply:
I meant like here: https://github.com/linkerd/linkerd2-proxy/blob/6ea4de5dce10c52d13cc1b1c34928a2036e14236/linkerd/app/integration/src/tests/telemetry.rs#L914

The various `test_whatever` functions add labels that are common between inbound and outbound to the `Labels` struct that's passed in, the idea was that the actual tests would only need to add the labels that _differ_ between inbound and outbound.